cmd/client/client.go

```diff
@@ -0,0 +1,216 @@
+// Copyright 2022 Jigsaw Operations LLC
```

Comment:
Could you rename `client.go` to `socks2ss.go`?
I think it's a lot clearer. There could be many different types clients. `client` is not clear.